sematic/ui/src/types/Types.tsx

```diff
@@ -318,6 +318,30 @@ function ListTypeView(props: TypeViewProps) {
   );
 }
 
+function UnionValueView(props: ValueViewProps) {
+  const typeRepr = props.typeRepr as AliasTypeRepr;
+
+  if (!typeRepr[2].args) {
```

Comment:
Since `typeRepr[2].args` appears several times, shall we assign it to a variable to give it a meaningful name?

e.g.
```
let [, , {args: typeArgs} ] = typeRepr
```

Reply:
Yup this sounds good. I will fix it in my next revision.